datadog/Tiltfile

```diff
@@ -105,22 +132,35 @@ def datadog_keys_secret():
     )
 
 
-def helm_resources():
+def helm_resources(agent_image_tag = None):
+    """Create Tilt resources required to install Datadog Agent from Helm charts.
+
+    Args:
+      agent_image_tag: Sets a specific image tag version for the Datadog Agent.
+    """
     helm_repo(
         "helm-datadog-repo",
         "https://helm.datadoghq.com",
         labels = ["datadog"],
     )
 
+    flags = []
+
     chart_values = os.path.join(extension_path, "values.yaml")
+    flags.append(["--values", chart_values])
+
+    if agent_image_tag:
+        flags.append(["--set", "agents.image.tag=%s" % agent_image_tag])
+        flags.append(["--set", "clusterAgent.image.tag=%s" % agent_image_tag])
+        flags.append(["--set", "clusterChecksRunner.image.tag=%s" % agent_image_tag])
```

Comment:
On startup:

```
Traceback (most recent call last):
  /Users/noelle/dev/DirectEmployers/MyJobs/Tiltfile:3:13: in <toplevel>
  <builtin>: in load_dynamic
  /Users/noelle/dev/DirectEmployers/MyJobs/apollo_gateway/Tiltfile:2:13: in <toplevel>
  <builtin>: in load_dynamic
  /Users/noelle/dev/DirectEmployers/MyJobs/partner_library_service/Tiltfile:1:13: in <toplevel>
  <builtin>: in load_dynamic
  /Users/noelle/dev/DirectEmployers/MyJobs/dev/Tiltfile:97:11: in <toplevel>
  /Users/noelle/Library/Application Support/tilt-dev/tilt_modules/github.com/DirectEmployers/tilt-extensions/datadog/Tiltfile:47:23: in datadog_up
  /Users/noelle/Library/Application Support/tilt-dev/tilt_modules/github.com/DirectEmployers/tilt-extensions/datadog/Tiltfile:159:18: in helm_resources
  /Users/noelle/Library/Application Support/tilt-dev/tilt_modules/github.com/tilt-dev/tilt-extensions/helm_resource/Tiltfile:113:20: in helm_resource
Error in k8s_custom_deploy: apply_cmd: a command must be a string or a list of strings: '["--values", "/Users/noelle/Library/Application Support/tilt-dev/tilt_modules/github.com/DirectEmployers/tilt-extensions/datadog/values.yaml"]' is a *starlark.List, not a string
```

You probably want to use [`extend()`](https://bazel.build/rules/lib/core/list#extend) instead of `append()`.

Reply:
:man_facepalming:

Reply:
Fixed